app/controllers/discover_controller.rb

```diff
@@ -1,3 +1,14 @@
 class DiscoverController < ApplicationController
-  def index; end
+  before_action :require_current_user
+
+  def index
+    # @films = FilmSearch.new.top_40_films if params[:button_clicked]
```

Comment:
If there is unused code we should remove it before opening a PR

Reply:
Thanks I got rid of it!